### ISSUE TITLE
Tailoring Typing Speed and Inter-Code Waiting Intervals

### ISSUE THREAD

**Issue by johannes-titz** [gaborcsardi/asciicast#54]
Impressive project!

Would it be possible to incorporate a parameter for adjusting typing speed and the waiting interval between "code chunks"? This feature could enable faster typing in specific sections and a more deliberate pace in others, catering to the need for additional time to comprehend specific outputs.

**Comment by gaborcsardi** [comment]
I am fairly sure that this is already possible, see around here: https://github.com/r-lib/asciicast/blob/main/R/asciicast.R#L7

**Comment by johannes-titz** [comment]
Alright, my apologies for any inconvenience caused. I appreciate your patience. I have another question: does this process require creating multiple recordings and then combining them? Is there a way to set waiting times and typing speeds before specific code segments, or am I overlooking a feature? For example:

```
#' Empty_wait: 1

some code

#' Empty_wait: 2

more code
```

This is not possible, right?

**Comment by gaborcsardi** [comment]
Yes, if you want different waiting times, then you need to merge casts with `merge_casts()`. You can insert `empty_wait` waits with empty lines, or with 
```
#! --
```
lines.

**Comment by gaborcsardi** [comment]
I think there is nothing to do here for me now?
